theme/themes/eea/globals/site.variables

```diff
@@ -194,9 +194,11 @@
        Links
 --------------------*/
 
-@linkColor           : @primaryColor;
+@linkColor           : @blue-4;
+@linkHoverColor      : @blue-5;
+@linkVisitedColor    : @blue-3;
```

Comment:
visited color is not accessible, I've added a comment in the ticket and we should get a new proposal from Manthos